lib/voice/VoiceConnection.js

```diff
@@ -314,7 +320,13 @@ class VoiceConnection extends EventEmitter {
                             this.disconnect(err);
                         }
                     });
-                    const udpMessage = Buffer.allocUnsafe(74);
+                    this.udpTimeout = setTimeout(() => {
+                        if(!this.ready) {
+                            this.disconnect(new Error("UDP response timeout"));
```

Comment:
I think it would be better to be more specific about what's wrong
```suggestion
                            this.disconnect(new Error("IP discovery timeout"));
```